framerail/src/routes/[slug]/[...extra]/+server.ts

```diff
@@ -0,0 +1,17 @@
+import { pageEdit } from "$lib/server/deepwell/edit.ts"
+
+export async function POST(event) {
+  let data = await event.request.formData()
+
+  let page = parseInt(data.get("page-id")?.toString() ?? "1")
+  let siteId = parseInt(data.get("site-id")?.toString() ?? "1")
+  let comments = data.get("comments")?.toString() ?? ""
+  let wikitext = data.get("wikitext")?.toString()
+  let title = data.get("title")?.toString()
+  let altTitle = data.get("alt-title")?.toString()
+  let tags = data.get("tags")?.toString().split(" ") ?? []
```

Comment:
The "space splitting" convention is not something we should copy from Wikidot on the server side. It's fine for now as a _client-side_ means of specifying tags, but on the server it should just be dealing with a JSON list of strings. This way we get rid of ambiguities around tag separations and can eventually have a better tag picker UI.

Reply:
Yeah this is related to the tag input in the simple makeshift editor being just a single line of input. When the editor gets some more work this should be changed.

Reply:
Oh you're right. I am too tired, I thought this was in deepwell. This is fine for now.